### ISSUE TITLE
Spalte "Anzahl Einträge" auch in der Datenpunktliste

### ISSUE THREAD

**Issue by martinrieder** [mdzio/ccu-historian#419]
In der Datenpunktkonfiguration wird eine Spalte für die Anzahl der Einträge gezeigt, nach der man auch sortieren kann. Wenn man aber die Werkzeuge aufrufen möchte, um Datenpunkte zu löschen, dann muss man erst zur Datenpunktliste wechseln und diese auswählen. Dieses Vorgehen ist recht umständlich und man muss sich dabei merken, welche Datenpunkte man bearbeiten möchte. Es wäre daher sehr von Vorteil, wenn diese Spalte auch in der Datenpunktliste vorhanden wäre.

**Comment by mdzio** [comment]
Den Rechtschreibfehler habe ich korrigiert.

Das Abfragen der Anzahl der Einträge je Datenpunkt dauert bei einer Installation als Add-On auf der langsamen CCU sehr lange. Daher müsste es per Konfigurationseintrag aktivierbar sein, wenn die Funktionalität gewollt ist.

Ich habe aber selber gerade zu wenig Zeit dies umzusetzen.

**Comment by martinrieder** [comment]
Ja, ich habe es auf der CCU laufen und das dauert für beides Datenpunktliste und - Konfiguration ungefähr gleich lang zu laden. Deshalb erwarte ich keine große Verschlechterung in der Performance.
Bei dauern 670 Datenpunkte ungefähr 20 Sekunden zum Laden. Das ist nervig, aber noch akzeptabel. Ich schaue mal rein, ob ich einen PR erstellen kann.

**Comment by mdzio** [comment]
Danke für das Lob.

PageRenderer.groovy gibt die Ladezeit jeder Seite im Log an:
```groovy
long st=System.currentTimeMillis()
...
log.finest "Page build time: ${System.currentTimeMillis()-st} ms"
```

Sicherlich könnte das auch auf einzelne SQL-Abfragen angewendet werden.

Für die Ermittlung der Anzahl der Einträge muss auf die Zeitreihentabelle des Datenpunktes zugegriffen werden. Dies ist bei der normalen Auflistung nicht nötig.

Ich sehe gerade ein ähnliches Feature in der _index.gy_ das bereits umgesetzt worden ist: Die Anzeige des letzten Zeitstempels.
Das erfolgt mit folgenden Zeilen:
```groovy
						if (webServer.config.showLastValue) {
							def last=database.getLast(dp)
							td class:'hidden-xs hidden-sm', utils.format(last?.timestamp)
							td class:'hidden-xs hidden-sm text-right', utils.format(last?.value)
						}
```

Dafür gibt es die Konfigurationsoption: webServer.showLastValue. Das könnte auch für die Anzahl der Einträge realisiert werden.
